cypress/e2e/task/createEditDeleteSubGradFinishRestoreTask.feature

```diff
@@ -9,18 +9,19 @@ Feature: Task - To create, edit and delete tasks by the teacher.
     Given I am logged in as a 'teacher1' at 'brb'
     When I go to rooms overview
     When I go to room 'Course with subject and tasks'
-    And I click on FAB to create new content
-    And I click on New Task FAB
+    When I click on FAB to create new content
+    When I click on New Task FAB
     Then I can see create task page '-'
-    And file upload button is disabled
+    Then file upload button is disabled
```

Comment:
The file upload button is not really a button it is a drop zone or?
I think it should not disabled anymore or? @bischofmax  @SevenWaysDP

Reply:
Ich passe das an.